foundry.toml

```diff
@@ -1,13 +1,58 @@
+# Full reference https://github.com/foundry-rs/foundry/tree/master/config
+
 [profile.default]
-src = "src"
-out = "out"
-libs = ["lib"]
-verbosity = 3
-solc_version = '0.8.20'
+  auto_detect_solc = false
+  block_number = 19537700 # Mar-29-2024 05:52:47 AM +UTC
+  bytecode_hash = "none"
+  cbor_metadata = false
+  fuzz = { runs = 1_000 }
+  gas_reports = ["*"]
+  libs = ["lib"]
+  optimizer = true
+  optimizer_runs = 10_000
+  out = "out"
+  script = "script"
+  solc = "0.8.20"
```

Comment:
we can use the latest solc but still choose if we wnat backwards compatibility through evm_version

Reply:
the key `solc` id deprecated afaik, use `solc_version` now